src/routes/components/Callout.tsx

```diff
@@ -109,6 +109,11 @@ export default function CalloutPage() {
       lang: "angular",
       description: "Sets the data-testid attribute. Used with ByTestId queries in tests.",
     },
+    {
```

Comment:
This whole thing needs a bit more. You need an entry for Angular, that uses `arialive`, and an entry for React that uses `ariaLive`. You need to set the `defaultValue` of `off`. And change the description to `"Indicates how assistive technology should handle updates to the live region"`, which is the same description used for our Notification Banner.

Reply:
PR updated